src/Parser/CameraCalibrationParser.js

```diff
@@ -21,17 +21,22 @@ THREE.Matrix4.prototype.setMatrix3 = function setMatrix3(m) {
     return this;
 };
 
+const textureLoader = new THREE.TextureLoader();
 const matrix3 = new THREE.Matrix3();
 // the json format encodes the following transformation:
 // extrinsics: p_local = rotation * (p_world - position)
 // intrinsics: p_pixel = projection * p_local
 // distortion: p_raw = distortion(p_pixel)
-function parseCalibration(calibration, options) {
+function parseCalibration(calibration, options = {}) {
+    const useMask = options.useMask == undefined ? true : options.useMask;
```

Comment:
This causes error in `orientation_utils` examples: please update the config of this example, upload the masks or change the default option here.

Reply:
done